src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/System.Text.RegularExpressions.Tests.csproj

```diff
@@ -11,7 +11,7 @@
     <IsHighAotMemoryUsageTest>true</IsHighAotMemoryUsageTest> <!-- to avoid OOMs with source generation in wasm: https://github.com/dotnet/runtime/pull/60701 -->
 
     <!-- Remove once the repo moves to a sufficiently high-enough version for file-scoped types -->
-    <MicrosoftCodeAnalysisVersion>4.4.0-1.22356.23</MicrosoftCodeAnalysisVersion>
+    <MicrosoftCodeAnalysisVersion Condition="'$(DotNetBuildFromSource)' != 'true'">4.4.0-1.22356.23</MicrosoftCodeAnalysisVersion>
```

Comment:
We got file-scoped types with .NET 7. Is this property still needed?

Reply:
It shouldn't be needed anymore.